pkg/eks/components/Config.vue

```diff
@@ -146,6 +157,23 @@ export default defineComponent({
   computed: {
     ...mapGetters({ t: 'i18n/t' }),
 
+    // the control plane k8s version can't be more than one minor version ahead of any node pools
+    // verify that all nodepools are on the same version as the control plane before showing upgrade optiopns
+    canUpgrade(): boolean {
+      if (this.mode === _CREATE) {
+        return true;
+      }
+      const nodeGroups = this.config?.nodeGroups || [];
+
+      const needsUpgrade = nodeGroups.filter((group) => semver.gt(semver.coerce(this.originalVersion), semver.coerce(group.version)) || group._isUpgrading);
```

Comment:
Out of curiosity, shouldn't we consider equal versions to be allowed and use `semver.gte`?

Reply:
If the node group version is equal to the cluster version, the node group can't be upgraded further (yet). Users need to upgrade the cluster version, then come back and upgrade the node group version. In this context we just care that node groups are not behind the current cluster version: if they were, attempting to upgrade the cluster further would make them more than one minor release apart which is not permitted by EKS

Reply:
Thanks for explaining